.travis.yml

```diff
@@ -10,7 +10,7 @@ cache: yarn
 # Run the the validate script
 # Temporarily also run the build script to make sure it works
 # (will move this to the release step once that's implemented)
-script: yarn run validate && yarn run build
+script: yarn run validate --ci && yarn run build
```

Comment:
now that you've reorganized this totally works! nice!

Reply:
that and the fact that yarn passes flags through, made this part really simple.